mutt/file.h

```diff
@@ -35,6 +35,18 @@ extern char *Tmpdir;
 #define MUTT_CONT (1 << 0) /**< \-continuation */
 #define MUTT_EOL  (1 << 1) /**< don't strip `\n` / `\r\n` */
 
+/* State record for mutt_file_iter_line() */
+struct mutt_file_iter {
+  char* s;                      /**< the line data */
```

Comment:
Wouldn't it be more understandable if `s` was named `line` and `line` was named `line_num`?

It'd make `mutt/file.c:726` easier to understand.

Reply:
These names were copied literally from the existing function mutt_file_read_line().

If I change mine, they need to be changed there too.  To me consistency in an API is even more important than easy readability of names.